dev-vcs/git-cola/git-cola-4.8.2.ebuild

```
@@ -44,6 +44,8 @@ distutils_enable_tests pytest
 
 src_prepare() {
 	sed -i "s|doc/git-cola =|doc/${PF} =|" setup.cfg || die
+	# remove bundled qtpy and polib
+	rm -Rf "${S}"/qtpy "${S}"/cola/polib.py || die
```

```suggestion
	rm -Rf qtpy cola/polib.py || die
```

`src_prepare` is run in `S`.